cpp/velox/udf/UdfLoader.cc

```diff
@@ -96,11 +99,17 @@ std::unordered_set<std::shared_ptr<UdfLoader::UdfSignature>> UdfLoader::getRegis
 
     // Handle UDAFs.
     void* getNumUdafSym = loadSymFromLibrary(handle, libPath, GLUTEN_TOSTRING(GLUTEN_GET_NUM_UDAF), false);
+    if (getNumUdafSym == nullptr) {
+      throw gluten::GlutenException("malloc failed");
+    }
```

Comment:
Does this change cause [this CI failure](https://github.com/apache/incubator-gluten/actions/runs/11220332046/job/31188265190?pr=7419)? 

```
org.apache.gluten.expression.VeloxUdfSuiteLocal *** ABORTED ***
  org.apache.gluten.exception.GlutenException: malloc failed
  at org.apache.gluten.udf.UdfJniWrapper.registerFunctionSignatures(Native Method)
  at org.apache.gluten.backendsapi.velox.VeloxListenerApi.onDriverStart(VeloxListenerApi.scala:92)
  at org.apache.gluten.backendsapi.SubstraitBackend.onDriverStart(SubstraitBackend.scala:27)
  at org.apache.gluten.backendsapi.SubstraitBackend.onDriverStart$(SubstraitBackend.scala:26)
  at org.apache.gluten.backendsapi.velox.VeloxBackend.onDriverStart(VeloxBackend.scala:52)
  at org.apache.gluten.GlutenDriverPlugin.init(GlutenPlugin.scala:65)
  at org.apache.spark.internal.plugin.DriverPluginContainer.$anonfun$driverPlugins$1(PluginContainer.scala:53)
  at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:293)
  at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
  at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
  ...
```

Reply:
It does! This check is not needed.